style.css

```diff
@@ -136,13 +136,18 @@ body {
   color: #1c1a19;
 }
 
+.closer img {
+  margin-left: 85%;
+}
+
 .social img {
   height: 20px;
 }
 
 .navigation img {
-  margin-left: auto;
```

Comment:
- [ ] Let's bring down the closed menu button and make sure it aligns on the same line as the popup menu project name (Profesional Art Printing Data)

| Your Design | Figma Design |
| ------------- | ------------- |
| ![18](https://user-images.githubusercontent.com/79058364/197249275-4f6d6284-4abb-471e-88f2-f1ad65eb41ae.png) | <img width="209" alt="image" src="https://user-images.githubusercontent.com/52934128/197008501-ca4d288b-a067-4369-95c3-f88b772d7c6c.png">  |